docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md

```diff
@@ -0,0 +1,49 @@
+# Aerosol optical properties/MACv2-SP
+
+## Summary
```

Comment:
Should we include something on Revision History

Reply:
Yes probably. I think we should capture that in the dataset view of our database so I wouldn't duplicate that here, but having a link to that resource is a good idea.

Reply:
Ok I've thought about it a bit more. It's a really good idea, but I'm going to need to do an hour of playing around to make it maintainable and easy I think. Let's see if I find that hour tomorrow or Friday.